scripts/crontab

```diff
@@ -9,9 +9,9 @@ SHELL=/bin/bash
 
 ######## cPouta production
 #  m  h  dom  mon  dow   command
-   0  0  1,16 *    *     source $HOME/bin/openrc_cpouta_prod_v3.sh; $HOME/diskimage-builder-scripts.cpouta/scripts/image_create_centos7.sh >> $HOME/log/c/centos7.log 2>&1
+#  0  0  1,16 *    *     source $HOME/bin/openrc_cpouta_prod_v3.sh; $HOME/diskimage-builder-scripts.cpouta/scripts/image_create_centos7.sh >> $HOME/log/c/centos7.log 2>&1
```

Comment:
Instead of commenting these out just like this, you could comment them out and mark them "Free slot"
Also all of the "Reserved for CentOS-*" should me marked free

Reply:
Those slots have been marked as "Free slot"